Autopilot/PathManager/Src/CommWithAttitudeManager.cpp

```diff
@@ -4,18 +4,23 @@
  */
 
 #include "CommWithAttitudeManager.hpp"
+#include "CommWithPathManager.hpp"
 
 extern "C"
 {
 #include "cmsis_os.h"
 }
 
-void CommWithAMInit()
+//Set up a mail queue for sending commands to the attitude manager
+osMailQDef(commandsMailQ, PATH_ATTITUDE_MAIL_Q_SIZE, CommandsForAM);
+osMailQId commandsMailQ;
```

Comment:
the name of the queue and the id are the same? Is this on purpose ? It's a bit confusing.

Reply:
I could rename the second one to commandsMailQID but I find this creates confusion on when to use commandsMailQ and when to use commandsMailQID as opposed to using a single name.

Reply:
I see what you mean, but to any code readers, this looks like you're using a variable before instantiating it. I would prefer using different names.